apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -201,6 +218,20 @@ private void sendMessage(JSONObject json) {
     log.debug(String.format("Sending Message: {} to {}", msg, TOPIC_ARN));
     PublishResult publishResult = snsClient.publish(publishRequest);
     // TODO: check on size of message and truncation etc (this can come later if/when we add more)
-    log.debug("Published SNS Message - " + publishResult.getMessageId());
+    log.info("Published SNS Message - " + publishResult.getMessageId());
```

Comment:
We should keep this on debug, in general operation we don't want to see all the messages in the logs.

Reply:
In Shunting Yard emitter, a similar log was being done for every mesasge. These messages come from hcatalog-server, so, putting the hcatalog-server in debug mode is not very straightforward for anyone who connects to shared hive metastore. Thats why, I think just having one line in the log that the emitter is actually sending the message is helpful. I am OK to revert if you still think otherwise.

Reply:
I don't have a strong opinion, I'd defer to people who look into these log files more than me - @rpoluri or @pradeepbhadani ?